package.json

```diff
@@ -1,10 +1,18 @@
 {
   "private": true,
   "scripts": {
-    "build:css": "sass sass:css",
-    "watch:css": "sass -w sass:css"
+    "watch": "NODE_ENV=production postcss ./sass/*.scss -d ./css --ext css -w",
+    "css": "NODE_ENV=production postcss ./sass/*.scss -d ./css --ext css"
   },
   "devDependencies": {
-    "sass": "^1.0.0"
+    "autoprefixer": "^10.4.13",
+    "cssnano": "^5.1.15",
+    "postcss": "^8.4.21",
+    "postcss-cli": "^9.1.0",
+    "postcss-import": "^14.1.0",
+    "postcss-loader": "^6.2.1",
+    "postcss-nesting": "^13.0.0",
+    "postcss-prefix-selector": "^1.16.1",
+    "postcss-simple-vars": "^6.0.3"
```

Comment:
This seems like a lot, do we need all of those?
How about just these to keep the dependencies light?
```
    "postcss": "^8.0.0",
    "postcss-cli": "^9.0.0",
    "autoprefixer": "^10.0.0",
    "postcss-prefix-selector": "^1.0.0",
    "postcss-preset-env": "^7.0.0"
```

Reply:
Yes, agreed. `cssnano` and `postcss-loader` can definitely go (didn't mean to include the loader). 

Not a fan of nesting whether natively supported or not, so `postcss-nesting` can for sure go.

`postcss-simple-vars` can easily be replaced by css native variables, so no need for that.

`postcss-import` is really only useful if you break out your css files into components or other subsets, which I don't think is needed.

So I'm good with all of those suggestions.